app/src/api/skills.js

```diff
@@ -87,8 +87,7 @@ export const updateSkillDetails = async (skillId, skillName, skillDesc, isActive
     }
     throw new Error("No data returned from backend");
   } catch (error) {
-    console.log(error);
-    return [];
+    return error.response.data;
```

Comment:
Will there be a case where err.res.data does not exist?

Reply:
removed FE changes as jon will be handling them in separate ticket instead